packages/fxa-settings/src/pages/ResetPassword/ConfirmResetPassword/index.tsx

```diff
@@ -49,7 +49,7 @@ const ConfirmResetPassword = (_: RouteComponentProps) => {
   }, [navigate]);
 
   if (!email || !passwordForgotToken) {
-    navigateToPasswordReset();
+    // navigateToPasswordReset();
```

Comment:
Is this commented out because there's a problem with getting the email from location state? If so, we might want to add a comment/TODO to fix?

Reply:
oh LORD. No that should not be in there anymore 😮‍💨 Will remove!